add alicloud

osswangxining opened the pull request:
Preview snapshot:

![image](https://user-images.githubusercontent.com/10258572/43565167-5d5b2c66-965c-11e8-9eb3-87044e23ab69.png)

![image](https://user-images.githubusercontent.com/10258572/43565188-735a1a18-965c-11e8-89f0-1a7026fc9f31.png)

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio.github.io/pull/2125#" title="Author self-approved">osswangxining</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **rcaballeromx**

If they are not already assigned, you can assign the PR to them by writing `/assign @rcaballeromx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[content/OWNERS](https://github.com/istio/istio.github.io/blob/master/content/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["rcaballeromx"]} -->

istio-testing commented:
Hi @osswangxining. Thanks for your PR.

I'm waiting for a [istio](https://github.com/orgs/istio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/istio/people) to skip this step.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

rootsongjc commented:
@osswangxining CI failed but have no matter with the cotent, you can make a new commit to trigger CI again.

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -0,0 +1,82 @@
+
+The Helm chart also offers significant customization options per individual service. Customize these per-service options at your own risk. The per-service options are exposed via the Parameters tab.
+
+Wait until Istio is fully deployed. Note that this can take up to several minutes.
```
I would make this a note:
{{< idea_icon >}} Before moving on, wait until Istio is fully deployed. Deployment can take up to several minutes.

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -0,0 +1,82 @@
+
+## What's next
+
+You can further explore the Istio functionality by following any of the tutorials in the [Guides](/docs/guides/) section. However, to do this you need to install `istioctl` to interact with Istio.
```
Simplify for clarity:
To further explore the Istio functionality, follow any of the tutorials in the [Guides](/docs/guides/) section. Before you do, install `istioctl` to interact with Istio.

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -0,0 +1,82 @@
+
+You can further explore the Istio functionality by following any of the tutorials in the [Guides](/docs/guides/) section. However, to do this you need to install `istioctl` to interact with Istio.
+
+The articles in this [series](https://yq.aliyun.com/articles/599874) provide detailed instructions about how to use Istio on AliCloud Kubernetes Container Service.
```
Edit for clarity and to improve content flow:
Next, you can follow the detailed instructions on [how to use Istio on the AliCloud Kubernetes Container Service](https://yq.aliyun.com/articles/599874).

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -0,0 +1,82 @@
+
+## Uninstalling
+
+1. Navigate to the Release section of the AliCloud Container Service console at [https://cs.console.aliyun.com/#/k8s/release/list](https://cs.console.aliyun.com/#/k8s/release/list)
```
1. Visit [the Release section of the AliCloud Container Service console](https://cs.console.aliyun.com/#/k8s/release/list).

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -0,0 +1,82 @@
+
+1. Navigate to the Release section of the AliCloud Container Service console at [https://cs.console.aliyun.com/#/k8s/release/list](https://cs.console.aliyun.com/#/k8s/release/list)
+
+1. Select the Release and click **Delete**. This will remove all the deployed Istio artifacts.
```
Split into separate steps for clarity. I edited to provide more clarity.
1. Select the release where you wish to uninstall Istio.
1. Click the **Delete** button to remove all the deployed Istio artifacts.

osswangxining commented on content/docs/setup/kubernetes/platform-setup/alicloud/index.md:
```diff
@@ -0,0 +1,38 @@
+
+Follow these instructions to prepare an AliCloud Kubernetes cluster for Istio.
+
+You can deploy a Kubernetes cluster to AliCloud quickly and easily in the Container Service console, which fully supports Istio.
```
updated all in new commit, please help review thanks a lot

osswangxining commented:
/assign @rcaballeromx   I submitted one commit including your comments at /pull/2125#pullrequestreview-142997704, can you help review again? Thank you very much!!!

rcaballeromx commented on content/docs/setup/kubernetes/platform-setup/alicloud/index.md:
```diff
@@ -9,30 +9,44 @@ keywords: [platform-setup,alicloud,aliyun]
+CIDR.
 
-1. See the step by step details from [the instructions](https://www.alibabacloud.com/help/doc-detail/53752.htm).
+1. Below is showing the GUI where all the previous steps are being performed.
```
this is not a numbered step, since the reader doesn't need to take any action. I suggest:
    The image below shows the GUI where you complete all the previous steps:
Consider moving this sentence and the image to before the steps. If you do, you should change the text accordingly:
    The image below shows the GUI where you complete all the following steps:

osswangxining replied:
see the new commit 4ab4781

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -5,49 +5,57 @@ weight: 12
-application.  It uses Application Catalog to automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/) for Kubernetes Engine.
+This guide installs the current release version of Istio and deploys the 
+[Bookinfo](/docs/examples/bookinfo/) sample application.  
+To automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/)
```
This sentence is incomplete. There is a missing action. What does the reader need to do to automate ...?

osswangxining replied:
agree, remove this wordy sentence

osswangxining replied:
see the new commit 4ab4781

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -5,49 +5,57 @@ weight: 12
+Kubernetes cluster quickly and easily in the `Container Service console`.
 
-- Make sure that `kubectl` can work fine for your Kubernetes cluster.
+- `kubectl` works fine for your Kubernetes cluster.
```
- Ensure `kubectl` works fine for your Kubernetes cluster.

osswangxining replied:
see the new commit 4ab4781

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -5,49 +5,57 @@ weight: 12
+ creates the `istio-system` namespace:
 
 {{< text bash >}}
     $ kubectl create namespace istio-system
```
Make sure the commands have the same indentation as the {{< text bash >}} markup. Otherwise, the linter will complain.

osswangxining replied:
see the new commit 4ab4781

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -5,49 +5,57 @@ weight: 12
 {{< /text >}}
 
-## Setup
+## Deploy Istio via Application Catalog
```
... via the Application Catalog

osswangxining replied:
see the new commit 4ab4781

rcaballeromx commented on content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md:
```diff
@@ -65,18 +73,26 @@ The Helm chart ships with reasonable default configuration options which are exp
 | `galley.enabled` | Specifies whether Galley should be installed for server-side config validation. Requires k8s >= 1.9 | `false` |
 
-The Helm chart also offers significant customization options per individual service. Customize these per-service options at your own risk. The per-service options are exposed via the Parameters tab.
+Passive voice issue:
```
Do not add the fact that there was a passive voice issue to the document. Be careful when copy-pasting :smiley:

osswangxining replied:
:-)

osswangxining replied:
see the new commit 4ab4781

osswangxining commented:
/assign @rcaballeromx  Thanks a lot for your great comments I submitted new version and can you help review again? Thank you very much!!! BTW You can see the preview form https://myistio.netlify.com/docs/setup/kubernetes/quick-start-alicloud-ack/

rcaballeromx commented:
You just need to address the CI/Linter errors.
They include removing trailing whitespaces and adding Alibaba to the .spelling file for it not to be seen as a spelling error.

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **linsun**

If they are not already assigned, you can assign the PR to them by writing `/assign @linsun` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/istio/istio.github.io/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["linsun"]} -->

istio-testing commented:
@osswangxining: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

osswangxining commented:
/assign @rcaballeromx Thanks a lot for your great help.  I have removed trailing whitespaces and added some terms into .spelling file in new commit. But in latest check, it looks that there are some files cannot be passed but these files are NOT submitted by me. Other check is interesting as below. Can you help check this one? I did not find how to fix it. Thanks a lot. 

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
./public/docs/setup/kubernetes/quick-start-alicloud-ack/index.html:11:                MARKDOWN ERROR: page description doesn't end with a period
